### PR TITLE
Blocks Single Content Card

### PR DESCRIPTION
**Pull request by Shubham-Sharma-77**
Define blocks single content card molecule

### CODE REVIEW

File: stories/Molecules/Blocks/SingleContent/singlecontent.scss [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  animation-duration: 300ms;
+  animation-timing-function: ease-out;
+  border: 0;
+  border-top: 2px solid $color-black;
+  display: block;
+  margin-bottom: $spacing-24;
+
+  & .singlecard__head {
+    font-size: 12px;
+    font-weight: 700;
```
Comment (surbhi90) [inline stories/Molecules/Blocks/SingleContent/singlecontent.scss:14]: @Shubham-Sharma-77 why we are using font weight

File: stories/Molecules/Blocks/SingleContent/singlecontent.scss [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    letter-spacing: 0.03em;
```
Comment (surbhi90) [inline stories/Molecules/Blocks/SingleContent/singlecontent.scss:15]: @Shubham-Sharma-77 can we give the letter spacing in px

File: stories/Molecules/Blocks/SingleContent/singlecontent.scss [hunk 3]
```diff
@@ -0,0 +1,71 @@
+@import '../../../assets/scss/variables';
+@import '../../../assets/scss/mixins';
+
+.singlecard {
+  animation-duration: 300ms;
+  animation-timing-function: ease-out;
+  border: 0;
+  border-top: 2px solid $color-black;
+  display: block;
+  margin-bottom: $spacing-24;
```
Comment (surbhi90) [inline stories/Molecules/Blocks/SingleContent/singlecontent.scss:10]: @Shubham-Sharma-77 why we are using animation

File: stories/Molecules/Blocks/SingleContent/singlecontent.scss [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    letter-spacing: 0.03em;
+    line-height: 13.5px;
+    padding: $spacing-16 $spacing-24;
+    text-transform: uppercase;
+  }
+
+  &__caption {
+    padding: $spacing-16 $spacing-24 $spacing-24;
+
+    & a.cta__link {
```
Comment (surbhi90) [inline stories/Molecules/Blocks/SingleContent/singlecontent.scss:24]: @Shubham-Sharma-77 why we are using tag with class name

File: stories/Molecules/Blocks/SingleContent/singlecontent.scss [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+  &__para {
+    display: block;
+    font-size: $font-size-16;
+    letter-spacing: 0;
+    line-height: 22px;
+  }
+
+  &-space {
+    a.cta__link {
```
Comment (surbhi90) [inline stories/Molecules/Blocks/SingleContent/singlecontent.scss:37]: @Shubham-Sharma-77 Why we are using tag with class name

File: stories/Molecules/Blocks/SingleContent/SingleContent.jsx [hunk 6]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import './singlecontent.scss';
+import '../../../assets/scss/_grid.scss';
+import { Heading } from '../../../Atom/Base-typography/Heading/Heading';
+import { Ctaspace } from '../../../Atom/Buttons-and-labels/Cta_button/cta_link_space';
+
+export const SingleContent = ({ contenttile, contentname, contentnametwo, image, type, paragraph,link,button }) => {
+    return (
+        <div className='grid-x grid-margin-x'>
+            <div className='cell large-4 small-12'>
```
Comment (surbhi90) [inline stories/Molecules/Blocks/SingleContent/SingleContent.jsx:10]: @Shubham-Sharma-77 We have to give medium-4 instead of large-4

**Review by surbhi90 — CHANGES_REQUESTED** [review]
Please check the comments

File: stories/Molecules/Blocks/SingleContent/singlecontent.scss (hunk 2 above)
Comment (Shubham-Sharma-77) [reply stories/Molecules/Blocks/SingleContent/singlecontent.scss:15]: @surbhi90 em are using in Figma.

**Review by surbhi90 — APPROVED** [review]

**Pull request merged**